base/src/main/java/com/smartdevicelink/managers/screen/TextAndGraphicUpdateOperation.java

```diff
@@ -221,7 +270,7 @@ Show assembleShowText(Show show) {
             return show;
         }
 
-        int numberOfLines = defaultMainWindowCapability != null ? ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(defaultMainWindowCapability) : 4;
+        int numberOfLines = defaultMainWindowCapability == null || shouldUpdateTemplateConfig() ? 4 : ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(defaultMainWindowCapability);
```

Comment:
We are just sending everything if shouldUpdateTemplateConfig is true because we don't know the capabilities of the new template at this time, when the capabilities change in the TextAndGraphicsManager it will send another update and correct anything that could be wrong